src/views/CreateAccount/CreateAccount.js

```diff
@@ -54,15 +56,27 @@ export default class CreateAccount extends React.Component {
     this.setState({ confirmPassword: e.target.value });
   }
 
+  getErrorMessage() {
+    if (this.state.error !== "") {
```

Comment:
@rrsampathi can we do do `if(this.state.error)` which will return false if `error` is `null, undefined, ""` etc?

Reply:
Yup that should work.